tests/acceptance/test_performance_trends.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import absolute_import
+
+import pytz
+
+from six.moves.urllib.parse import urlencode
+from mock import patch
+
+from django.db.models import F
+from sentry.models import Project
+from sentry.testutils import AcceptanceTestCase, SnubaTestCase
+from sentry.testutils.helpers.datetime import before_now, iso_format
+from sentry.utils.samples import load_data
+
+from .page_objects.base import BasePage
+
+FEATURE_NAMES = ("organizations:trends", "organizations:performance-view",)
+
+
+def make_nth_transaction(base_event, name, n, start, end):
+    event = base_event.copy()
+
+    event["transaction"] = name
+    event["event_id"] = "{:02x}".format(n).rjust(32, '0')
+    event["start_timestamp"] = iso_format(start)
+    event["timestamp"] = iso_format(end)
+
+    return event
+
+
+def make_trend(store_event, project_id, event, name, first_duration, second_duration, number_transactions=2, period_mins=60):
+    for i in range(number_transactions):
+        time_between = period_mins / number_transactions
+        minutes = period_mins - ((i + 1) * time_between) + (time_between / 2)
+        if (i < (number_transactions / 2)):
```

Comment:
```suggestion
        if i < (number_transactions / 2):
```
Don't really need the extra parens here 💅